osu.Game/Overlays/Settings/Sections/Input/TabletSettings.cs

```diff
@@ -0,0 +1,268 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Input.Handlers.Tablet;
+using osu.Framework.Platform;
+using osu.Framework.Threading;
+using osu.Game.Graphics.Sprites;
+using osuTK;
+
+namespace osu.Game.Overlays.Settings.Sections.Input
+{
+    public class TabletSettings : SettingsSubsection
+    {
+        private readonly ITabletHandler tabletHandler;
+
+        private readonly Bindable<Vector2> areaOffset = new Bindable<Vector2>();
+        private readonly Bindable<Vector2> areaSize = new Bindable<Vector2>();
+        private readonly IBindable<TabletInfo> tablet = new Bindable<TabletInfo>();
+
+        private readonly BindableNumber<float> offsetX = new BindableNumber<float> { MinValue = 0 };
+        private readonly BindableNumber<float> offsetY = new BindableNumber<float> { MinValue = 0 };
+
+        private readonly BindableNumber<float> sizeX = new BindableNumber<float> { MinValue = 10 };
+        private readonly BindableNumber<float> sizeY = new BindableNumber<float> { MinValue = 10 };
+
+        [Resolved]
+        private GameHost host { get; set; }
+
+        /// <summary>
+        /// Based on the longest available smartphone.
+        /// </summary>
+        private const float largest_feasible_aspect_ratio = 20f / 9;
```

Comment:
Ultrawide PC configurations are commonly 21:9, so I'd like at least support up to that.

```suggestion
        private const float largest_feasible_aspect_ratio = 21f / 9;
```

Reply:
Fair enough. Comment needs to be updated too in that case.